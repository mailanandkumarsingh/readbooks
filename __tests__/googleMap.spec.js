import React from 'react';
import Chart from '../src/components/chart';
import GoogleMap from '../src/components/google_map';
import renderer from 'react-test-renderer';

describe('Chart Rendering', () => {

  it('renders empty BarChart component correctly', () => {
    const map = renderer.create(
      <GoogleMap lon={-0.1277} lat={51.5073} />
      //Passing an array of 40 elements similar to what is being 
      // returned by the 5 days weather app. This is a snapshot capture
    ).toJSON();
    expect(map).toMatchSnapshot();
  });

});