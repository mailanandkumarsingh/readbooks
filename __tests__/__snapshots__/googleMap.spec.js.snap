// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Chart Rendering renders empty BarChart component correctly 1`] = `
<div
  style={
    Object {
      "height": "100%",
    }
  }
/>
`;
